doc/architecture.rst

```diff
@@ -103,17 +97,18 @@ forth.
 Scalability and High Availability
 ---------------------------------
 
-In traditional architectures, clients talk to a centralized component (e.g., a
-gateway, broker, API, facade, etc.), which acts as a single point of entry to a
-complex subsystem. This imposes a limit to both performance and scalability,
-while introducing a single point of failure (i.e., if the centralized component
-goes down, the whole system goes down, too).
+In traditional architectures, clients talk to a centralized component. This
+centralized component might be a gateway, a broker, an API, or a facade. A
+centralized component of this kind acts as a single point of entry to a complex
+subsystem. Architectures that rely upon this kind of centralized component
+incur a limit to performance and scalability and have a single point of
+failure. If the centralized component goes down, the whole system goes down.
 
-Ceph eliminates the centralized gateway to enable clients to interact with 
-Ceph OSD Daemons directly. Ceph OSD Daemons create object replicas on other
-Ceph Nodes to ensure data safety and high availability. Ceph also uses a cluster
-of monitors to ensure high availability. To eliminate centralization, Ceph 
-uses an algorithm called CRUSH.
+Ceph eliminates this centralized component. This enables clients to interact
+with Ceph OSD Daemons directly. Ceph OSD Daemons create object replicas on
```

Comment:
We have the age-old "OSD" vs "OSD Daemons".  I'll always favor the former, but you're the boss.

Reply:
My preference has now been ratified by your preference. So: our preference will go into the docs.